docs/next/guides/upgrade-and-migration/release-notes.md

```diff
@@ -13,6 +13,34 @@ tags:
 
 [[toc]]
 
+## 10.0.0
+
+Released on 29th of September, 2021
+
+**Changed**
+- **Breaking change** Unified the naming and description of the fourth argument, `controller`, for selection manipulation in the [`beforeOnCellMouseDown`](@/api/hooks.md#beforeoncellmousedown) and [`beforeOnCellMouseOver`](@/api/hooks.md#beforeoncellmouseover) hooks. [#4996](https://github.com/handsontable/handsontable/issues/4996)
+- **Breaking change** Changed what the [`beforeRender`](@/api/hooks.md#beforerender) and [`afterRender`](@/api/hooks.md#afterrender) hooks are, and when they are triggered. Added two new hooks: `beforeViewRender` and `afterViewRender`. [#6303](https://github.com/handsontable/handsontable/issues/6303)
```

Comment:
I think the links would be useful for these new-new hooks. This will be consistent with the rest of the links in the text and make it easier to read.

Suggested ready-made links:

 [`beforeViewRender`](@/api/hooks.md#beforeviewrender)
 [`afterViewRender`](@/api/hooks.md#afterviewrender)

Reply:
@aninde - sure, done: 48fb94a